### PR TITLE
CNV Database - bug fixes

### PR DESCRIPTION
**Pull request by SophieS9**
- Fix to calculating classification based on score. Was previously not working if 0.9 or -0.9. Added unit tests for this which pass. 
- Fixed typo in view CNV classification page where the words "First check" were used for the second check score.

### CODE REVIEW

File: acmg_db/utils/cnv_utils.py [hunk 1]
```diff
@@ -269,16 +269,20 @@ def calculate_acmg_class(score):
 	print(score)
```
Comment (josephhalstead) [inline acmg_db/utils/cnv_utils.py]: get rid of print statement

File: acmg_db/utils/cnv_utils.py [hunk 2]
```diff
@@ -269,16 +269,20 @@ def calculate_acmg_class(score):
...
+			return "4"
+		elif 0.90 <= score <= 0.98: 
+			return "3"
+		elif -(0.89) <= score <= 0.89:
+			return "2"
+		elif -(0.98) <= score <= -(0.90):
+			return "1"
+		elif score <= -(0.99):
+			return "0"
+		else:
```
Comment (josephhalstead) [inline acmg_db/utils/cnv_utils.py:283]: can we also have the function return something e.g. return "error" 0r "-1"

**Review by josephhalstead — CHANGES_REQUESTED** [review]
just a few small changes requested

File: acmg_db/utils/cnv_utils.py (hunk 1 above)
Comment (SophieS9) [reply acmg_db/utils/cnv_utils.py]: Fixed in https://github.com/AWGL/variant_classification_DB/pull/60/commits/aa56f2968168203e67261b3362e89b6e52d1497e

File: acmg_db/utils/cnv_utils.py (hunk 2 above)
Comment (SophieS9) [reply acmg_db/utils/cnv_utils.py:283]: Fixed in https://github.com/AWGL/variant_classification_DB/pull/60/commits/1374cbb9393975dea203a2562009af6f6de326aa

**Review by josephhalstead — APPROVED** [review]

**Pull request merged**